### PR TITLE
(nginx) Add /ServiceAccount package parameter

### PR DESCRIPTION
**Pull request by flcdrg**
## Description

- Also includes 1.17.9 (missing from repo, but already published)

Implements #48

## Motivation and Context
Allow nginx to run under lower-privilege service accounts

## How Has this Been Tested?
Tested locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [x] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/mkevenaar/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

File: automatic/nginx/README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ nginx [engine x] is an HTTP and reverse proxy server, a mail proxy server, a gen
 * `/serviceName` - The name of the windows service which will be create. Default: `nginx`
 * `/port` - The port Nginx will listen to. Default: `80`
 * `/noService` - Don't install the nginx windows service
+* `/serviceAccount` - account to run Windows Service. One of `System`, `LocalService` or `NetworkService`. Default: `System`
```
Comment (mkevenaar) [inline automatic/nginx/README.md]: Could you tell me where you have found these parameter values? In the documentation I only find username/pass for `ObjectName`

File: automatic/nginx/tools/chocolateyInstall.ps1 [hunk 2]
```diff
@@ -5,10 +5,11 @@ $pp = Get-PackageParameters
 
 $arguments = @{
     packageName = $env:chocolateyPackageName
-    file        = "$toolsDir\nginx-1.17.8.zip"
+    file        = "$toolsDir\nginx-1.17.9.zip"
     destination = if ($pp.installLocation) { $pp.installLocation } else { Get-ToolsLocation }
     port        = if ($pp.Port) { $pp.Port } else { 80 }
     serviceName = if ($pp.NoService) { $null } elseif ($pp.serviceName) { $pp.serviceName } else { 'nginx' }
+    serviceAccount = if ($pp.ServiceAccount) { $pp.ServiceAccount } else { 'SYSTEM' }
```
[the remark this answers is not in the document]
Comment (flcdrg) [reply automatic/nginx/tools/chocolateyInstall.ps1]: According to the nssm docs, System and LocalSystem are synonyms.. https://github.com/kirillkovalenko/nssm/blob/master/README.txt#L809

File: automatic/nginx/tools/helpers.ps1 [hunk 3]
```diff
@@ -86,6 +86,7 @@ function Install-NginxService {
     nssm set $($arguments.serviceName) AppDirectory  "$($nginxPaths.NginxDir)" 2>&1 | Out-Null
     nssm set $($arguments.serviceName) AppNoConsole 1 2>&1 | Out-Null # Mentioned on top off http://nssm.cc/download
     nssm set $($arguments.serviceName) AppStopMethodSkip 7 2>&1 | Out-Null
+    nssm set $($arguments.serviceName) ObjectName "$($arguments.ServiceAccount)" 2>&1 | Out-Null
```
[the remark this answers is not in the document]
Comment (flcdrg) [reply automatic/nginx/tools/helpers.ps1]: That's true, except for service accounts (which don't have a known password). cf https://github.com/kirillkovalenko/nssm/blob/master/README.txt#L806

File: automatic/nginx/README.md (hunk 1 above)
Comment (flcdrg) [reply automatic/nginx/README.md]: From this list: https://github.com/kirillkovalenko/nssm/blob/master/README.txt#L809

**Comment by mkevenaar** [comment]
@flcdrg your changes have been merged and will be used on the next release. Thank you for your contribution!

**Pull request merged**